src/deviation.jl

```diff
@@ -82,15 +116,35 @@ function gkldiv{T<:AbstractFloat}(a::AbstractArray{T}, b::AbstractArray{T})
 end
 
 # MeanAD: mean absolute deviation
+"""
+    meanad(a::AbstractArray{T}, b::AbstractArray{T})
+
+Computes the mean absolute deviation between `a` and `b`, i.e. `mean(abs(a - b))`.
```

Comment:
I these docstring and those below are really the model we should follow: without a redundant list of arguments, they give all the information one needs.